plugins/nv-filters/nvvfx-load.h

```diff
@@ -247,6 +247,12 @@ typedef enum NvCVImage_ComponentType {
 #define NVVFX_STATE "State"          //!< State variable
 #define NVVFX_STATE_SIZE "StateSize" //!< Number of bytes needed to store state
 
+/** Logging support **/
+#define NVCV_LOG_FATAL 0   //!< Message to be printed right before aborting due to an unrecoverable error.
+#define NVCV_LOG_ERROR 1   //!< An operation has failed, but it is not fatal.
+#define NVCV_LOG_WARNING 2 //!< Something was not quite right, but we fixed it up, perhaps at a loss in performance.
+#define NVCV_LOG_INFO 3    //!< Nothing is wrong, but this information might be of interest.
```

Comment:
Why are those defines and not an `enum`?

Reply:
Fixed